### PR TITLE
automation: add 4.4 branches automation

### PR DESCRIPTION
**Pull request by sandrobonazzola**
## Changes introduced with this PR

* Extend automation to 4.4 branches

## Are you the owner of the code you are sending in, or do you have permission of the owner?

[y/n] yes

### CODE REVIEW

File: .github/workflows/check-patch.yml [hunk 1]
```diff
@@ -41,9 +40,18 @@ jobs:
...
-    - name: test install
+    - name: install master built rpm
+      if: contains(github.ref, "ref/head/master")
       run: |
           yum install -y exported-artifacts/ovirt-release-master-4*noarch.rpm
+
+    - name: install 4.4 built rpm
+      if: contains(github.ref, "ref/head/ovirt-4\.4")
+      run: |
+          yum install -y exported-artifacts/ovirt-release44-4*noarch.rpm
```
Comment (janosdebugs) [inline .github/workflows/check-patch.yml:51]: Can we just install `exported-artifacts/*.rpm`? If that's not possible, please add a comment explaining the reasoning behind this logic.

File: .github/workflows/check-patch.yml [hunk 2]
```diff
@@ -59,11 +67,11 @@ jobs:
 
 
   build-el9:
-
+    # Build only on master
```
Comment (janosdebugs) [inline .github/workflows/check-patch.yml:70]: This comment is redundant as the code below already states as much. Please add a comment explaining the reasons behind this decision.

**Review by janosdebugs — CHANGES_REQUESTED** [review]

File: .github/workflows/check-patch.yml (hunk 1 above)
Comment (sandrobonazzola) [reply .github/workflows/check-patch.yml]: Added comment

File: .github/workflows/check-patch.yml (hunk 2 above)
Comment (sandrobonazzola) [reply .github/workflows/check-patch.yml]: fixed comment

**Review by janosdebugs — APPROVED** [review]

**Pull request merged**